src/utils.jl

```diff
@@ -214,3 +214,40 @@ macro matchable(expr)
         Base.length(x::$name) = $(length(fields) + 1)
     end |> esc
 end
+
+# Check for linear coefficients
+coefficients(expr,sym::SymbolicUtils.Sym) coefficients(expr,[sym])
+function coefficients(expr,syms::AbstractArray{SymbolicUtils.Sym})
```

Comment:
Please follow the coding style of the rest of the package. Add a space after a comma.